webapp/src/components/form/index.tsx

```diff
@@ -35,7 +35,7 @@ const Form = ({fieldConfig: {label, type, validations}, value, optionsList, onCh
             <Input
                 type='text'
                 placeholder={label}
-                value={value}
+                value={value ?? ''}
```

Comment:
rather than ?? i think we should use || 
```suggestion
                value={value || ''}
```

Reply:
we need to check only for `null/undefined` except that any value is valid here